docs/technical-design/howto-style-css-scss.md

```diff
@@ -0,0 +1,99 @@
+
+### Styles should be as narrowly scoped as possible
+
+This is 101 but easy to forget. Please read [this section](https://github.com/trussworks/Engineering-Playbook/blob/main/docs/web/frontend/developing-ui.md#style-with-css-modules) of the Truss eng playbook for a good summary.
+
+TL;DR Less is more with styles. In our application, the preferred way to style React code is via locally scoped [CSS module](https://github.com/css-modules/css-modules), written as a `<component>.module.scss` file and stored alongside React component files. We also use [CSS selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference#selectors) to further target styles. Tightly scoped styles and CSS [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) ensure separation of concerns.
+
+Example of scoped styles:
+- Shared styles live close to the workflow they apply to. For example, directories in  `/pages` folder may share a `*.module.scss` stylesheet if needed for templates in that area of the application. CSS selectors should be used to further scope down styles.
+- Re-useable atomic components (see `/components` folder) have self contained styles. There is a named `*.module.scss` file for each component.
+
+## How to import
+- Prefer namespaced scss imports with [`@use`](https://sass-lang.com/documentation/at-rules/use/) over [`@import`](https://sass-lang.com/documentation/at-rules/import/) for scss to reduce loading styles multiple times. Sass team is deprecating [`@import`](https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline).
+
+In `*.module.scss` files this looks like (at top of file):
+```
+@use '../../styles/custom.scss' as custom;
+@use '../../styles/uswdsImports.scss' as uswds;
+```
+
+and then referring to varibles and mixins inline with the [namespace](https://sass-lang.com/documentation/at-rules/use/#choosing-a-namespace) - such as `custom.$mcr-foundation-white` or  `@include uswds.uswds-at-media(tablet)`
+#### When to use global overrides
+
+Global styles live in a single file (`overrides.scss`). This should be used sparingly.
+
+Here are concrete examples where global style overrides are needed:
+    1. We are dealing with container elements that repeat throughout the application outside of a specific component (divs, cards, display tables)
+    2. We need override a behavior in USWDS that we can't control via settings throughout the application
+    3. We want to hide an element from the screen using our global screenreader only class
+
+
+Prefer encapsulating into components with their own module.scss.
+
+### React components may use conditional styles and compose together styles using the `classnames` package
+
+- Sometimes React components have complex logic related to styles. To assist with this use [`classnames`](https://github.com/JedWatson/classnames) to compose together styles from different places.
```

Comment:
Standardized on `classnames` to follow @trussworks/react-uswds. Wanted css patterns to be folks be well trodden path for folks less comfortable with it.

I don't recommend template literals unless wrapped in some kind of shared helper function that would check what comes out looks correct. Particularly when you have cases with multiple conditionals (4+)  - file by file template literals get messy. If we wanted to swap out classnames packages (very open to that) would rather make our own simple function pattern and use it everywhere in one refactor rather than file by file handling. 

@pearl-truss Would you want to see a sentence on this in the docs? Or else something like a quick basic ADR on this - could write easily.

Reply:
Gotcha, thanks for laying that out! I think adding a blurb to the docs here would be enough